frontend/src/views/MainMenu.vue

```diff
@@ -24,6 +33,33 @@
 
 <script setup>
 import Navbar from "@/components/Navbar.vue"
+import sound from "../assets/music_4.mp3"
+
+//const audioSource = new Audio("../assets/music_4.mp3");
+//const audioSource = new (sound);
+//audioSource.play();
+
+export default {
+  components: {
+    Navbar,
+    Leaderboard,
+  },
+
+  data() {
+    return {
+      musicPlays: false
+    }
+  },
+
+  methods: {
+    mp() {
```

Comment:
could you rename this method to be something more descriptive? I'm not sure what mp means.

Reply:
i think it might be for music playing but im not entirely sure since i wrote it quite a while ago

Reply:
it might be better to rename it something like `play()` or `startMusic()` in that case.